src/translations/lang/en.json

```diff
@@ -345,12 +345,14 @@
     "MEMBERSHIP_DROPDOWN_PLACEHOLDER": "Select membership plan",
     "MEMBERSHIP_ACCORDION_FIRST_CONTENT": "The MyCrypto Membership is a monthly subscription that gives you additional features and helps to support MyCrypto’s future developments.",
     "MEMBERSHIP_ACCORDION_FIRST_TITLE": "What is the MyCrypto Membership?",
-    "MEMBERSHIP_ACCORDION_FORTH_CONTENT": "As we continue to add features to the MyCrypto Membership, we may adjust the prices. However, if you have an active membership at the time we add new features, you will get access to those new features for no extra cost until your existing membership expires.",
-    "MEMBERSHIP_ACCORDION_FORTH_TITLE": "Will the price stay the same forever?",
+    "MEMBERSHIP_ACCORDION_FOURTH_CONTENT": "As we continue to add features to the MyCrypto Membership, we may adjust the prices. However, if you have an active membership at the time we add new features, you will get access to those new features for no extra cost until your existing membership expires.",
+    "MEMBERSHIP_ACCORDION_FOURTH_TITLE": "Will the price stay the same forever?",
     "MEMBERSHIP_ACCORDION_SECOND_CONTENT": "4 DAI (~$4.00) per month, or you can purchase a 12-month package at a discounted rate. For the hardcore, there’s also a lifetime membership.",
     "MEMBERSHIP_ACCORDION_SECOND_TITLE": "How much does the MyCrypto Membership cost?",
     "MEMBERSHIP_ACCORDION_THIRD_CONTENT": "We use the [Unlock Protocol](https://unlock-protocol.com/) for this. You’ll be purchasing a token that unlocks features of MyCrypto when we detect that you hold that token. Each token has an expiration date according to the duration of membership you purchased.",
     "MEMBERSHIP_ACCORDION_THIRD_TITLE": "How does the MyCrypto Membership work?",
+    "MEMBERSHIP_ACCORDION_FIFTH_CONTENT": "You can purchase a MyCrypto Membership with DAI and ETH, or on the xDai network with [xDai](https://support.mycrypto.com/general-knowledge/scaling/how-to-get-xdai). We will likely be adding support for more networks and tokens in the future.",
```

Comment:
This line reads wrong to me. Might just be me though 😆

Reply:
this is jordan-submitted copy. bring it up with the bossman xD

Reply:
Updated as per discussion w/ bossmang